cmd/commands/cli/command_cli.go

```diff
@@ -354,6 +355,24 @@ func (c *Command) identities(argsString string) {
 	}
 }
 
+func (c *Command) registration(argsString string) {
+	status, err := c.tequilapi.RegistrationStatus(argsString)
+	if err != nil {
+		warn("Something went wrong: ", err)
```

Comment:
we could implement new method here, like error(items ...interface{}), since its error, not warning.

Reply:
This sounds like another PR to improve CLI "ux". Let's register task for that